templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -4,6 +4,13 @@ contents:
   inline: |
     #!/bin/bash
     set -eux
+    # Workaround to ensure OVS is installed due to bug in systemd Requires:
+    # https://bugzilla.redhat.com/show_bug.cgi?id=1888017
+    if ! rpm -qa | grep -q openvswitch; then
```

Comment:
@trozet let's do `openvswitch2.13`

Reply:
maybe just use `rpm -qa openvswitch`  ? using grep here can also give a match for packages which has openvswitch as sub-string

Reply:
If we go for rpm -q, it would have to be `rpm -q openvswitch2.13` FWIW

Reply:
"using grep here can also give a match for packages which has openvswitch as sub-string"

Yeah that was intentional. What if we move to openvswitch2.14 and the package changes names? I don't think that's better. Really openvswitch package should not have the version in its package name, but to workaround this we only look for openvswitch here.

Reply:
sure, if this is preferred way by the OVN team :)